src/parser/BinaryTokenizer.h

```diff
@@ -9,6 +9,7 @@
 #ifndef SQUID_SRC_PARSER_BINARYTOKENIZER_H
 #define SQUID_SRC_PARSER_BINARYTOKENIZER_H
 
+#include "ip/Address.h"
```

Comment:
> We cannot just forward declare it because these two methods return the Ip::Address object

Naturally, return types do not require a full declaration (except where the return value is _used_, of course): https://stackoverflow.com/questions/553682/when-can-i-use-a-forward-declaration/553869#553869

Reply:
Done.